patterns/2-structured/misusing metrics.md

```diff
@@ -0,0 +1,48 @@
+## Title
+
+Misusing Metrics
+
+## Patlet
+
+Metrics can be misused causing friction across teams. In these situations, create a sense of ownership within teams on how to use the data, avoiding comparisons, and begin measuring progress with deltas.
+
+## Problem
+
+The existence of metrics may lead to undesired situations and frictions across teams. Gamifications, list of rock stars or any other way of having people or teams “better” than others usually brings into the discussion defensive behaviors where the metric needs to be deprecated.
```

Comment:
The "brings into the discussion defensive behaviors where the metric needs to be deprecated" sounds strange to me.

If you would split this up in multiple sentences, how would you phrase it?

Reply:
The existence of metrics may lead to undesired situations and frictions across teams. For example, implementing a gamifications approach which elevates some contributors or teams as "better" than others has a negative impact to the internal culture. Utilizing metrics as a weapon to indicate performance results in defensive behaviors or disengagement. These delay and inhibit successful and comprehensive understanding of the value of metrics. Ultimately, teams and individuals become weary of metrics all together. 

How is that? In retrospect, "metrics being deprecated" needs to be added to the solutions below.

Reply:
Negative consequences of metrics is a great thing to tackle. Thanks for writing this up. Gamification of metrics definitely leads to some strange (and sometimes unwanted) behaviors in organizations sometimes, i.e. "promotion driven development" as described inhttps://twitter.com/gergelyorosz/status/1442162670753431559?lang=en

Another lens to view this through, is that incentives, intentional or otherwise, almost always have second order effects that may be desirable or undesirable besides the intended consequences. I think Kent Beck's blog has some good writing about this, but I can't find a good reference online to share other than https://geekincentives.substack.com/p/the-geek-incentives-manifesto which is very brief.
 
I think it's important to note is that these unintended consequences can lead to negative outcomes in an organization even when the incentives had good intentions and the ones affected by them did as well. 

Using the "weapon" phrasing may imply that bad outcomes are intended by bad actors, whereas I think in reality, bad actions are often the unintended consequences of well meaning people, or at least people with an average amount of healthy-self interest in advancing their career or project, but in an organization there are always competing projects and other peers seeking similar outcomes that need the same resources (money, time, attention, budget, etc.) 

That's not to say that there are never bad actors, just that things actually can go wrong without requiring it as a precondition.

Reply:
Thanks for the references and calling out the word choice. I'll rephase.